packages/databases-collections/src/modules/create-collection/index.js

```diff
@@ -111,6 +113,16 @@ export const createCollection = (data) => {
           return stopWithError(dispatch, err);
         }
 
+        const trackEvent = {
+          is_capped: !!data.options.capped,
```

Comment:
should we add some guard to check if `data.options` is there, or by this time will always be there?

Reply:
I've checked all the usage and it seems to always be at least `{}`.